File: html-converter-dotnet-core-cs/EventHandlingExample.cs

using ceTe.DynamicPDF.HtmlConverter;
using System;

namespace html_converter_dotnet_core_cs
{
    internal class EventHandlingExample
    {
        public static void Run(string outputPath)
        {
            Converter.Loaded += OnPageLoaded;
            Converter.Convert(new Uri("https://www.google.com"), outputPath);
        }
        private static void OnPageLoaded(LoadedEventAgrs args)
        {
            Console.WriteLine("Page loaded.");
        }
    }
}

File: html-converter-dotnet-core-vb/EventHandlingExample.vb
Imports ceTe.DynamicPDF.HtmlConverter



Friend Class EventHandlingExample

    Public Shared Sub Run(outputPath As String)

        AddHandler Converter.Loaded, AddressOf OnPageLoaded

        Converter.Convert(
                New Uri("https://www.google.com"),
                outputPath)

    End Sub

    Private Shared Sub OnPageLoaded(
            args As LoadedEventAgrs)

        Console.WriteLine("Page loaded.")

    End Sub

End Class
